.eslintrc/.eslintrc.comments.eslintrc

```diff
@@ -2,6 +2,9 @@
   "plugins": ["eslint-plugin-eslint-comments"],
   "extends": ["plugin:eslint-comments/recommended"],
   "rules": {
-    "eslint-comments/no-use": ["error", { "allow": [] }]
+    "eslint-comments/no-use": [
+      "error",
+      { "allow": ["eslint-disable-next-line"] }
```

Comment:
we intend to disable this for people not abuse this inline disable.

Do you mind create a variable for null in high chart folder and disable rule in eslintrc under that package folder?

Reply:
Good idea! Will do.